### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Missing a backpack, added it back and moved the starting list to the correct file.
Fixing some other small issues
Started to remove medical borgs, to make sure the medical ships wont error out.

### CODE REVIEW

**Comment by Cheackraze** [comment]
Curious if this is what was causing the spawning bug

File: Resources/Maps/Shuttles/helix.yml [hunk 1]
```diff
@@ -2775,13 +2775,13 @@ entities:
...
-    - pos: 7.5,6.5
-      parent: 1
-      type: Transform
+#- proto: SpawnPointMedicalCyborg
+#  entities:
+#  - uid: 401
+#    components:
+#    - pos: 7.5,6.5
+#      parent: 1
+#      type: Transform
```
Comment (Cheackraze) [inline Resources/Maps/Shuttles/helix.yml:2784]: Does the map loader allow commented entities? Regardless the game code will bulldoze that uid etc.  Better to just yeet and delete the map entry tbh

File: Resources/Maps/Shuttles/pulse.yml [hunk 2]
```diff
@@ -1085,13 +1085,13 @@ entities:
...
-    - pos: 0.5,0.5
-      parent: 1
-      type: Transform
+#- proto: SpawnPointMedicalCyborg
+#  entities:
+#  - uid: 139
+#    components:
+#    - pos: 0.5,0.5
+#      parent: 1
+#      type: Transform
```
Comment (Cheackraze) [inline Resources/Maps/Shuttles/pulse.yml:1094]: Same here

**Comment by dvir001** [comment]
> Curious if this is what was causing the spawning bug

It seem it was, the spawn was there, but the job was out.

**Review by Cheackraze — CHANGES_REQUESTED** [review]

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**